src/wiktionary_scraper.py

```diff
@@ -0,0 +1,122 @@
+from bs4 import BeautifulSoup
+import os
+import re
+import requests
+import time
+
+
+def pronunciations_from_wiktionary_english(word):
```

Comment:
i think you could do a class instead of independent functions. that would make the code more encapsulated and more readable while making the functions of this module simpler to read too